src/plans/README.md

```diff
@@ -42,4 +42,22 @@ See the comments inside all the plans for more information.
    the ground, which creates joint over-torquing warnings and errors.
 
 6. Continuous: non-terminating plan that performs continuous operations, useful
-   as a stress/load test.
\ No newline at end of file
+   as a stress/load test.
+
+Testing plans
+-------------
+
+### ReferenceMission2 ###
+
+- How to run:
+	- Launch OceanWATERS ([Launch Tutorial](https://github.com/nasa/ow_simulator/wiki/Tutorials))
+	- Source your OceanWATERS workspace
+	- Run the autonomy node with this plan as the argument:
+	  `roslaunch ow_autonomy autonomy_node.launch plan:=ReferenceMission2.plx`
+
+- Success criteria: 
+	- Left running the plan completes
+		- Upon completion, the plan will print "ReferenceMission2 plan complete."
+		- To more easily view this message when it occurs, run the above roslaunch command followed by `grep -i "ReferenceMission2 plan complete."`
+	- The plan is interruptable through fault injection, i.e. pauses in response to a fault
+		- Faults can be injected via rqt, a python script, or the command line ([Fault Injection Tutorial](https://github.com/nasa/ow_simulator/blob/master/ow_faults/README.md))
```

Comment:
This is good, though as a rule I like Markdown files to view well in an ASCII text editor, and the one thing that would help here are shorter lines -- note that the rest of the file stays within 80 columns.

Reply:
No problem, I will rearrange it a bit to fit that format.